mix.exs

```diff
@@ -32,37 +33,45 @@ defmodule PhoenixStarter.MixProject do
   defp deps do
     [
       {:assert_identity, "~> 0.1.0", only: :test},
-      {:bamboo, "~> 1.6"},
-      {:bamboo_ses, "~> 0.1.0"},
       {:bcrypt_elixir, "~> 2.0"},
-      {:credo, "~> 1.5.0-rc.2", only: [:dev, :test], runtime: false},
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
+      {:dialyxir, "~> 1.0", only: [:dev, :test], runtime: false},
       {:ecto_psql_extras, "~> 0.2"},
       {:ecto_sql, "~> 3.4"},
       {:ex_aws_s3, "~> 2.1"},
       {:ex_doc, "~> 0.22", only: :dev, runtime: false},
       {:floki, ">= 0.27.0", only: :test},
+      {:gen_smtp, "~> 1.0"},
```

Comment:
I included it because of this, from the docs:

> This email adapter makes use of the Amazon SES SendRawEmail action and generates a SMTP style message containing the information to be emailed. This allows for greater more customizable email message and ensures the capability to add attachments. As a result, however, the :gen_smtp dependency is required in order to correctly generate the SMTP message that will be sent.

https://hexdocs.pm/swoosh/Swoosh.Adapters.AmazonSES.html